### DISCUSSION TITLE
Search results for a location sorted by distance to that location?

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#15616] [Q&A] [answered]
We got a ticket on support, claiming that sorting a location search result by distance to that location does **no longer** work but the results are shown with distance to your current position.

AFAIK we never had that feature but results are always shown with distance to your current position....but I might be wrong as I did not closely follow the development with utilizing the changed server features.

Is the initially described behavior a bug/regression or simply a feature we do not yet have?

**Comment by fm-sys** [comment, marked as answer]
Yes, it's a regression. I implemented that feature 3 years ago... See PR #11691
